R/Spectra-functions.R

```diff
@@ -80,7 +80,7 @@ addProcessing <- function(object, FUN, ...) {
 #'
 #' @noRd
 .peaksapply <- function(object, FUN = NULL, ..., f = dataStorage(object),
-                        BPPARAM = bpparam()) {
+                         BPPARAM = bpparam()) {
```

Comment:
Is this space intended?
```suggestion
                        BPPARAM = bpparam()) {
```